examples/tf/multi_env_trpo.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+"""This is an example to train multiple tasks with TRPO algorithm."""
+import gym
+import tensorflow as tf
+
+from garage.envs import normalize
+from garage.envs.multi_env_wrapper import MultiEnvWrapper
+from garage.experiment import run_experiment
+from garage.np.baselines import LinearFeatureBaseline
+from garage.tf.algos import TRPO
+from garage.tf.envs import TfEnv
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import CategoricalMLPPolicy
+
+
+def run_task(snapshot_config, *_):
+    """Run task.
+
+    Args:
+        snapshot_config (garage.experiment.SnapshotConfig): The snapshot
+            configuration used by LocalRunner to create the snapshotter.
+
+        _ (object): Ignored by this function.
+
+    """
+    with LocalTFRunner(snapshot_config=snapshot_config) as runner:
+        env1 = TfEnv(normalize(gym.make('Adventure-ram-v4')))
```

Comment:
maybe you could use PointEnv for these in the future, since this is unlikely to actually train to any solution...